src/TMedia.cpp

```diff
@@ -747,7 +747,7 @@ void TMedia::play(TMediaData& mediaData)
 
         pPlayer.getMediaPlayer()->setMedia(QUrl::fromLocalFile(absolutePathFileName));
     } else {
-        QMediaPlaylist* playlist = new QMediaPlaylist;
+        auto playlist = new QMediaPlaylist;
```

Comment:
I'd have preferred a name of the form `pPlaylist` - as I tend to lose track of whether it is a class instance or a pointer to such a thing... but then I think rain is wet so who am I to say? :man_shrugging: